### DISCUSSION TITLE
Production readyness / experience

### DISCUSSION THREAD

**Discussion by hmmmsausages** [ory/keto#1075] [Q&A] [answered]
Hi everyone, 👋 

Even though it's a bit of a vague question, I've been wondering about the production readyness of Keto. In particular after seeing that the ["Going to production" page](https://www.ory.sh/docs/keto/guides/production) still seems to be under development and marked with a red "Danger" callout box.
What's the general notion of the production readyness of Keto? Would welcome any experience of people running it in production and with applications that deal with several thousand subjects and objects and multitude of relations between them?

PS: I've had a quick look through previous discussions here, but couldn't find this being discussed before. Sorry if I missed it.

**Comment by vinckr** [comment, marked as answer]
Hello @hmmmsausages 
apologies for the late answer, the "Going to production" doc is intended as a guide for self-hosted Ory Keto instances, similar to the [guide](https://www.ory.sh/docs/hydra/self-hosted/production) we have for Hydra. 
Looking for contributions from the community for all self-hosting documentation - there are many different use cases and approaches out there and sharing the knowledge helps immensely

Ory Keto is ready for production - when you are hosting it yourself you need to take care of things like database and API gateway yourself. 
If your database can handle the traffic from "several thousand subjects and objects" then Keto will have no problem! Feel free to check out the [Ory Network](https://console.ory.sh/registration) if you don't want to deal with hosting/maintenance and just use the API, there are free dev accounts as well.

**Reply by hmmmsausages** [reply]
Thanks a lot @vinckr . That answers it for me
